Week1/queries.js

```diff
@@ -0,0 +1,48 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+  database: 'myWorld',
+});
+
+connection.connect();
+
+function makeQuery(query) {
```

Comment:
I would personally modify this query to show the user what is happening. I would change it in a way that the output looks like (for instance):

```
SELECT Name FROM countries WHERE Continent LIKE '%Europe%'                                                                                                                  
RowDataPacket { Name: 'Belgium' }                                                                                                                                           
RowDataPacket { Name: 'Germany' }                                                                                                                                           
RowDataPacket { Name: 'Russian Federation' }                                                                                                                                
RowDataPacket { Name: 'Spain' }                                                                                                                                             
RowDataPacket { Name: 'France' }                                                                                                                                            
RowDataPacket { Name: 'Turkey' }                                                                                                                                            
RowDataPacket { Name: 'Netherlands' }      
```